scico/linop/xray/_xray.py

```diff
@@ -94,14 +95,22 @@ def __init__(
         self.angles = angles
 
         self.nx = np.array(im_shape)
-
-        if x0 is None:
-            x0 = -self.nx / 2
-        self.x0 = x0
         if dx is None:
             dx = np.ones(2)
+        if is_scalar_equiv(dx):
+            dx = dx * np.ones(2)
         self.dx = dx
 
+        if np.sqrt(np.sum(dx**2)) > 2:
+            Warning(
```

Comment:
This just creates a `Warning` exception and does nothing with it. Add `import warnings` at the top of the module and use `warnings.warn(message)` here.